features/admin.application-templates.v1/models/endpoints.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Interface for the Application Templates Management feature resource endpoints.
+ */
+export interface ApplicationsTemplatesEndpointsInterface {
```

Comment:
Do we use Interface suffix in other places as well? if not, it'd be better to change the name to preserve consistency.

```suggestion
export interface ApplicationsTemplatesEndpoints {
```

Reply:
Normally we are using the interface part https://github.com/wso2/identity-apps/blob/05605b9367987c95ce06add63ce0e95494a90b2d/features/admin.applications.v1/models/endpoints.ts#L24